exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,18 @@
+function greetings(mentor) {
```

Comment:
if you re-read the instructions, it is asking for **2** functions here
1 to change the string to upper case
1 to create a shouty greeting
whilst what you have done does work, it isn't what the exercise is asking for